### DISCUSSION TITLE
Inquiry about the Release Time of Instructions or Tutorials for Using cuEquivariance Library for MACE Model in Lammps

### DISCUSSION THREAD

**Discussion by clecust** [ACEsuit/mace#731] [Q&A] [answered]
Dear developers,

I'm currently working on large-scale system simulations with the MACE model in Lammps. For my research, the simulation speed is of utmost importance and I'm highly sensitive to it. I've learned that the cuEquivariance library has the potential to accelerate the MACE model through CUDA in training model.
However, I've been searching for instructions or reference tutorials on how to use the cuEquivariance library for CUDA acceleration when applying the MACE model in Lammps, but unfortunately, I haven't been able to find any so far.

Therefore, I would like to enquire:
**Are there any plans to release instructions or reference tutorials for CUDA acceleration of MACE models in Lammps using the cuEquivariance library?**
If there are plans to release it, can you tell me the approximate timeframe so that I can rationalise my research schedule and preparations.

Thank you very much for your time and attention. I'm looking forward to your reply.
Best regards,

**Comment by gabor1** [comment, marked as answer]
The reason you didn't find the instructions is because currently cuEq doesn't support running within lammps. We are working with Nvidia to enable this feature, and it should happen in the next 1-2 months.

**Reply by clecust** [reply]
Very much looking forward to it! Thank you for your excellent contributions!
